core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.js

```diff
@@ -93,8 +96,24 @@ function toggleDagViz(forJob) {
     // Save the graph for later so we don't have to render it again
     graphContainer().style("display", "none");
   }
+
+  window.localStorage.setItem("expand-dag-viz-arrow-" + forJob, "" + status);
 }
 
+$(function (){
+  if (window.localStorage.getItem("expand-dag-viz-arrow-false") == "true") {
```

Comment:
it's not intuitive what `expand-dag-viz-arrow-false` means. I would expect something like `expand-dag-viz-arrow-stage-page` or something.